### ISSUE TITLE
async pipe

### ISSUE THREAD

**Issue by timkindberg** [ngUpgraders/ng-forward#73]
We should see if it's possible to write an async pipe (aka filter) for ng1 that supports promises and observables.

@robwormald any interest? :)

**Comment by dsebastien** [comment]
+1, I'm currently evaluating whether we could be using Observables w/ Angular 1 to ease our transition to Angular 2 later on. Unfortunately due to priorities we had to start with Angular 1 still :(

**Comment by robwormald** [comment]
i know @ggoodman was working on something along this lines, ping!

**Comment by ggoodman** [comment]
Yes, I put together https://github.com/ggoodman/angular-observe that takes a bit of a different approach to watching `Promise`s or `Observable`s (or plain values).

**Comment by timkindberg** [comment]
@ggoodman that's awesome. Would it be feasible to shove that all into an angular 1 'async' filter? Even if it needed to be reduced in scope.
